app/routes/people/person/positions/minister.js

```diff
@@ -0,0 +1,24 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class PeoplePersonPositionsMinisterRoute extends Route {
+  @service store;
+
+  async model() {
+    let person = this.modelFor('people.person');
+
+    let { positionId: ministerId } = this.paramsFor(
+      'people.person.positions.minister'
+    );
+
+    let minister = this.store.findRecord('minister', ministerId, {
+      reload: true,
+      include: ['contacts', 'minister-position'].join(),
```

Comment:
It seems there are still some async calls happening, so I would include those relationships as well:
![image](https://user-images.githubusercontent.com/3533236/128029035-f7620e94-f420-4cf8-939b-950965241651.png)

Reply:
Thank you, I just learned to check the network tab to discover the relationships that should be included!